pyccel/codegen/printing/ccode.py

```diff
@@ -586,6 +602,68 @@ def function_signature(self, expr):
         else:
             return '{0}{1}({2})'.format(ret_type, name, arg_code)
 
+    def _print_IndexedElement(self, expr):
+        if isinstance(expr.base, IndexedVariable):
+            base = expr.base.internal_variable
+        else:
+            base = expr.base
```

Comment:
This is not used but I think it should save you some checks later if you did use it

Reply:
i did copy it from the fcode i thought it will be useful to catch some checks